### DISCUSSION TITLE
Kibana/Elastic login not working (SO Version: 2.3.130)

### DISCUSSION THREAD

**Discussion by erwintwr2** [Security-Onion-Solutions/securityonion#8286] [General] [answered]
Hi SO Team

I am having a challenge (new install, production Standalone deployment), where i am unable to login to Kibana/Elastisearch.
if i open the Kibana Link provided on the left tab of the SOC, it does present the Elastisearch page.
Entering an invalid login gives me "Username or password is incorrect ,okease try again)
Entering a valid login (same as the one used on SOC), shows a loading elastic screen for ~2 seconds, then returns to the login page.

so-status shows all modules [OK] 

Is there any additional steps i can take to get this going?

Thank you!
Erwin

**Comment by dougburks** [comment, marked as answer]
What browser are you using?

Have you tried incognito mode or another browser altogether?

Have you checked the Kibana logs for clues?

**Reply by erwintwr2** [reply]
What browser are you using? -> Chrome Version 103.0.5060.114

Have you tried incognito mode or another browser altogether? Tested both now. Incognito and Firefox works fine

Have you checked the Kibana logs for clues?

/opt/so/log/kibana/kibana.log
`{"ecs":{"version":"1.12.0"},"@timestamp":"2022-07-14T14:00:59.273+00:00","message":"Logging in with provider \"basic\" (basic)","log":{"level":"INFO","logger":"plugins.security.routes"},"process":{"pid":295}}
`

 docker logs so-kibana ( from hitting the login button)
https://pastebin.com/jtGHJ1BK

Not sure what to look for (statuscodes is all 200)

Any other ideas?
One odd part -> firefox immediately requested me to save my user/password for the SOC, and auto completed it to the Kibana login page. My chrome does not request this (and i didn't check the "never save" option

**Reply by erwintwr2** [reply]
Clearing my chrome cookies specifically for the SOC url corrected the issue
